src/SIM/SIMbase.C

```diff
@@ -1256,7 +1256,7 @@ bool SIMbase::solutionNorms (const TimeDomain& time,
   gNorm.resize(norm->getNoFields(0));
   size_t nNorms = 0;
   for (i = 0; i < gNorm.size(); i++)
-    if (i == 0 || i > ssol.size() || !ssol[i-1].empty())
+    if (i == 0 || i >= ssol.size())
```

Comment:
As discussed, I have to think more about the consequence of this change before merging